src/wazuh_db/schema_upgrade_v8.sql

```diff
@@ -0,0 +1,22 @@
+/*
+ * SQL Schema for upgrading databases
+ * Copyright (C) 2015-2020, Wazuh Inc.
```

Comment:
We are un 2021!
However, please delete this file.

Reply:
Fixed